### PR TITLE
Fix github urls for upload and pom metadata

### PR DESCRIPTION
**Pull request by lefou**
This should fix the release process.

I think a review is essential here. The CI will not help much until we merge into master and see a release happen.

### CODE REVIEW

File: build.sc [hunk 1]
```diff
@@ -662,12 +667,12 @@ def publishExecutable() = {
...
     if (!unstable){
       requests.post(
-        "https://api.github.com/repos/lihaoyi/Ammonite/releases",
+        s"https://api.github.com/repos/${ghOrg}/${ghRepo}/releases",
         data = ujson.write(
           ujson.Obj(
             "tag_name" -> buildVersion,
             "name" -> buildVersion,
-            "body" -> s"http://www.lihaoyi.com/Ammonite/#$buildVersion"
+            "body" -> s"${homePage}/#${buildVersion}"
```
Comment (sake92) [inline build.sc:675]: Will this have double slash? Because `homePage` has a trailing slash.  
I prefer to have base URLs without a trailing slash..

Comment (lefou) [reply build.sc:675]: Yeah, makes sense. Me too.

File: build.sc [hunk 2]
```diff
@@ -4,9 +4,14 @@ import $file.ci.upload
 
 import $ivy.`io.get-coursier::coursier-launcher:2.0.0-RC6-10`
 
+val ghOrg = "com-lihaoyi"
+val ghRepo = "Ammonite"
+val masterBranch = "master"
+val homePage = "http://ammonite.io/"
```
Comment (sake92) [inline build.sc]: https? And without trailing slash please. 😄

**Review by sake92 — APPROVED** [review]
LGTM

**Review by alexarchambault — APPROVED** [review]
Let's see how it goes!

**Pull request merged**